src/worker/util/achievement.ts

```diff
@@ -131,17 +131,29 @@ async function getAll(): Promise<
 
 const check = async (when: AchievementWhen, conditions: Conditions) => {
 	try {
+		/* took out this early return to explicitly let users know when achievements arent awarded.
 		if (g.get("easyDifficultyInPast") || g.get("godModeInPast")) {
 			return;
-		}
+		}*/
+
+		const tooEz = g.get("easyDifficultyInPast") || g.get("godModeInPast");
 
 		const awarded: string[] = [];
 
 		for (const achievement of achievements) {
 			if (achievement.when === when && achievement.check !== undefined) {
 				const result = await achievement.check();
 
-				if (result) {
+				if (result && tooEz) {
+					logEvent(
+						{
+							type: "achievement",
+							text: `"${achievement.name}" achievement not awarded due to Easy/God mode. <a href="/account">View all achievements.</a>`,
```

Comment:
Might as well tell them exactly why. One of these, depending on the scenario...

"${achievement.name}" achievement not awarded because ....

* God Mode is enabled.

* God Mode was enabled in the past.

* the difficulty level is Easy.

* the difficulty level was previously Easy.

For checking the current difficulty level, `g.get("difficulty") <= DIFFICULTY.Easy`. And God Mode's current status is the boolean `g.get("godMode")`.